### DISCUSSION TITLE
Biome support in Zed | Biome is a fast formatter for JavaScript, TypeScript, JSX, and JSON that scores 97% compatibility with Prettier, saving CI and developer time.

### DISCUSSION THREAD

**Discussion by a4arpon** [zed-industries/zed#7978] [Feature Requests]
Does zed has BiomeJS support? Recently I am using it in my production app and it perform very well and provide very high performance then Prettier and Eslint together. I think implementing Biome with Zed will be very great. 
![image](https://github.com/zed-industries/zed/assets/70208852/70687354-c921-4556-9363-8b6b641749a6)

**Comment by SomeoneToIgnore** [comment]
Zed does not have biome integration, so far there's only an open issue for supporting it: https://github.com/zed-industries/zed/issues/4402
